### PR TITLE
[SMALLFIX] Fixed Javadoc in BlockLockManagerTest

### PR DESCRIPTION
**Pull request by HorizonNet**
Fixed Javadoc in the *BlockLockManagerTest* class.

### CODE REVIEW

File: core/server/src/test/java/alluxio/worker/block/BlockLockManagerTest.java [hunk 1]
```diff
@@ -231,7 +231,8 @@ public void dontReuseLockTest() throws Exception {
    * Calls {@link BlockLockManager#lockBlock(long, long, BlockLockType)} and fails if it doesn't
    * hang.
    *
-   * @param the block id to try locking
+   * @param manager the manager to used to lock
```
Comment (aaudiber) [inline core/server/src/test/java/alluxio/worker/block/BlockLockManagerTest.java]: NIT: the manager to call lock on

**Comment by aaudiber** [comment]
LGTM, just left a minor comment

**Comment by HorizonNet** [comment]
@aaudiber Thanks for the comment. I updated the PR accordingly.

**Comment by aaudiber** [comment]
Thanks, LGTM

**Pull request merged**